src/value.rs

```diff
@@ -93,6 +97,20 @@ value_impl_from! {
     )
 }
 
+#[cfg(feature = "num-bigint")]
+impl From<i64> for JsValue {
```

Comment:
Probably will also want a `From<u64/i128/u128>`

Reply:
It is added now